include/barebox.mk

```diff
@@ -0,0 +1,102 @@
+PKG_NAME ?= barebox
+
+ifndef PKG_SOURCE_PROTO
+PKG_SOURCE = $(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL = \
+	http://sources.lede-project.org \
+	http://www.barebox.org/download
+endif
+
+PKG_BUILD_DIR = $(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_TARGETS := bin
```

Comment:
I'm not sure whether this is needed.

Reply:
This is needed, or multiple barebox packages will be used in the same build directory.